### DISCUSSION TITLE
moadl alert design going wrong

### DISCUSSION THREAD

**Discussion by ayemin-tun** [filamentphp/filament#11404] [General]
### Description 

why my alert modal show like this I make composer update and also npm update but not work

![image](https://github.com/filamentphp/filament/assets/76741291/97acf5a5-dde8-42f6-8db6-36502969064c)

**Comment by leandrocfe** [comment]
run `npm run build`

**Reply by ayemin-tun** [reply]
it not work my project don't run npm run dev or build it only write with laravel filament 

```
$ npm run dev

> dev
> vite

'vite' is not recognized as an internal or external command,
operable program or batch file.
```

**Reply by leandrocfe** [reply]
https://discord.com/channels/883083792112300104/1204854710981627914/1204854710981627914
